### ISSUE TITLE
Javascript exceptions from SignatureToolTipManager.java cluttering logs

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#2274]
### System details

    RStudio Edition : Server
    RStudio Version : 1.1.419
    OS Version      : Linux / Chrome
    R Version       : 3.4.1

### Steps to reproduce the problem

Via a support investigation, received a customer log. It contains hundreds of consecutive instances of the following Javascript exception from SignatureTooltipManager.java, in addition to some more potentially "interesting" errors.

### Describe the problem in detail

Lots of noise makes it harder to spot real problems. Plus there's a non-zero chance that a barrage of Javascript exceptions is upsetting the browser and/or our logging system.

### Describe the behavior you expected

Common Javascript null exceptions, ideally, should be fixed so they aren't causing all this noise. There have been fixes in 1.2 to this source file specifically to deal with null references; perhaps worth backporting.

```
19 Feb 2018 15:40:57 [rserver] CLIENT EXCEPTION (rsession-user): (TypeError) : Cannot read property 'row' of undefined;|||org/rstudio/studio/client/workbench/views/source/editors/text/r/SignatureToolTipManager.java#134::execute|||com/google/gwt/core/client/impl/SchedulerImpl.java#167::runScheduledTasks|||com/google/gwt/core/client/impl/SchedulerImpl.java#338::flushPostEventPumpCommands|||com/google/gwt/core/client/impl/SchedulerImpl.java#76::execute|||com/google/gwt/core/client/impl/SchedulerImpl.java#140::execute|||com/google/gwt/core/client/impl/Impl.java#244::apply|||com/google/gwt/core/client/impl/Impl.java#283::entry0|||rstudio-0.js#-1::eval|||com/google/gwt/cell/client/AbstractEditableCell.java#41::viewDataMap|||Client-ID: 4b6d3f72-3d0a-47d0-a66b-fd80860180f7|||User-Agent: Mozilla/5.0 (Macintosh  Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36
```

**Comment by emcrisan** [comment]
Logs full of this error on https://rstudioide.zendesk.com/agent/tickets/34369.  
RSP version 1.1.463.
R version: 3.5.1
OS Version: Redhat 7.6

**Comment by gtritchie** [comment]
This was fixed in 1.2, but won't be back-ported to 1.1, so closing now.
